adding details, minor corrections, ToBeDiscussed items, clarification…

allenjbaum opened the pull request:
… and more

lots of little changes

jeremybennett commented:
Hi Allen,

Sorry for the slow reply - I've been out of circulation. This is good, but needs an entry in the ChangeLog file to indicate what you have changed.  Something like:
```
2019-04-04  Allen Baum  <allen.baum@esperantotech.com>

	Adding details, minor corrections, ToBeDiscussed items and
	clarifications to the specification of the future compliance test
	suite.

	* spec/TestFormatSpec.adoc: Updated with more detail, corrections
	and clarifications.
```
Your commit message should then be something like:
```
Update specification of future compliance test suite.

	Adding details, minor corrections, ToBeDiscussed items and
	clarifications.

ChangeLog:

	* spec/TestFormatSpec.adoc: Updated with more detail, corrections
	and clarifications.
```
Which meets git commit message standards.

The rationale is that a ChangeLog can be reviewed _before_ the patch is approved and committed.

allenjbaum commented:
I am a complete nepophyte with git; i'll need a lot of hand holding.

so, I've added this to the changelog:
2019-04-05 Allen Baum <allen.baum@esperantotech.com>
    * spec/TestFormatSpec.adoc:  Adding details, minor corrections,
ToBeDiscussed
      items and clarifications to the specification of the future
compliance test
      suite. Also removing restrictions on having absolute addresses in
signature

I'm not making (further) changes to the spec.adoc itself.

Then, I'll do the same thing to the changelog for the misalign test changes
(in another branch or repository - I still don't have all this figured out)
I'll try to figure out how to add and commit this...

On Thu, Apr 4, 2019 at 12:03 PM Jeremy Bennett <notifications@github.com>
wrote:

> Hi Allen,
>
> Sorry for the slow reply - I've been out of circulation. This is good, but
> needs an entry in the ChangeLog file to indicate what you have changed.
> Something like:
>
> 2019-04-04  Allen Baum  <allen.baum@esperantotech.com>
>
> 	Adding details, minor corrections, ToBeDiscussed items and
> 	clarifications to the specification of the future compliance test
> 	suite.
>
> 	* spec/TestFormatSpec.adoc: Updated with more detail, corrections
> 	and clarifications.
>
> Your commit message should then be something like:
>
> Update specification of future compliance test suite.
>
> 	Adding details, minor corrections, ToBeDiscussed items and
> 	clarifications.
>
> ChangeLog:
>
> 	* spec/TestFormatSpec.adoc: Updated with more detail, corrections
> 	and clarifications.
>
> Which meets git commit message standards.
>
> The rationale is that a ChangeLog can be reviewed *before* the patch is
> approved and committed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/riscv/riscv-compliance/pull/53#issuecomment-480023739>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Ad96plafaDQbjnilbOCs_o350oZ7hrCEks5vdkyWgaJpZM4cAMx5>
> .
>
